clicker.py

```diff
@@ -127,7 +123,7 @@ def toggle_event(key):
             print(Fore.GREEN + 'Started Manual Mode\n')
         
             cps = int(input(Fore.WHITE + "Enter CPS: \n"))
-            speed_ = 1/cps
+            speed_ = round(1/(cps+ cps*0.15),4)

```

Comment:
can you make a new pull request with just this? Other things i've fixed already in newest commit.